### PR TITLE
2 examples of small data training in medical imaging

### PR DESCRIPTION
**Pull request by alxndrkalinin**
Following [the suggestion](https://github.com/greenelab/deep-review/pull/824#issuecomment-369744402), very short addition of 2 examples in Medical Imaging:
- TL + augmentations on very small data
- positive mining with no ground truth

### CODE REVIEW

File: content/03.categorize.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ In more recent work, Wang et al. [@tag:Wang2016_breast_cancer] analyzed stained
 On this task a pathologist has about a 3% error rate.
 The pathologist did not produce any false positives, but did have a number of false negatives.
 The algorithm had about twice the error rate of a pathologist, but the errors were not strongly correlated.
+Combining pre-trained deep network architectures with multiple augmentation techniques, including different rotations, image scales, crop sizes, and image normalization methods enables accurate detection of breast cancer from a very small set of histology images with less than a 100 images per class [@tag:Rakhlin2018_histology].
```
Comment (agitter) [inline content/03.categorize.md]: Could possibly cut the phrase `, including different rotations, image scales, crop sizes, and image normalization methods` to make this even more compact.

Also `less than a 100` should be `less than 100`.

File: content/03.categorize.md [hunk 2]
```diff
@@ -104,6 +105,12 @@ The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consist
 Further, Wang et al. [@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework, to detect common thoracic diseases.
 It showed superior performance over a benchmark using fully-labeled data.
 
+Another example of semi-automated label generation for hand radiograph segmentation employed positive mining, an iterative procedure that combines manual labeling with automatic processing [@tag:Iglovikov2017_baa].
+First, the initial training set was created by manually labeling 100 of 12,600 unlabeled radiographs that were used to train a model and predict labels for the rest of the dataset.
+Then, through manual inspection, poor quality predictions were discarded, while the initial training set was expanded with the acceptable segmentations, and the process was repeated.
```
Comment (agitter) [inline content/03.categorize.md]: There are many commas here.  How about `Then, poor quality predictions were discarded through manual inspection,  the initial training set was expanded with the acceptable segmentations, and the process was repeated.`

File: content/03.categorize.md [hunk 3]
```diff
@@ -104,6 +105,12 @@ The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consist
...
+This procedure had to be repeated 6 times to obtain good quality segmentation labeling for all radiographs, except for a last 100 corner cases that still required manual annotation.
```
Comment (agitter) [inline content/03.categorize.md]: `6` -> `six`

This reads better if you remove `a last`

File: content/03.categorize.md (hunk 1 above)
Comment (alxndrkalinin) [reply content/03.categorize.md]: Done

File: content/03.categorize.md (hunk 2 above)
Comment (alxndrkalinin) [reply content/03.categorize.md]: Yes, changed, thanks

File: content/03.categorize.md (hunk 3 above)
Comment (alxndrkalinin) [reply content/03.categorize.md]: Done

**Comment by alxndrkalinin** [comment]
@agitter thanks for the useful suggestions – it reads better now!
